Utilities/ParaView/export-scene-macro.py

```diff
@@ -430,12 +440,13 @@ def mkdir_p(path):
     renProp = renProps.GetItemAsObject(rpIdx)
     if not renProp.GetVisibility():
       continue
+
     if hasattr(renProp, 'GetMapper'):
       mapper = renProp.GetMapper()
-      dataObject = mapper.GetInputDataObject(0, 0);
+      dataObject = mapper.GetInputDataObject(0, 0)
       dataset = None
 
-      if dataObject.IsA('vtkCompositeDataSet'):
+      if dataObject.IsA('vtkCompositeDataSet') or dataObject.IsA('vtkMultiBlockDataSet'):
```

Comment:
`vtkMultiBlockDataSet` is actually a `vtkCompositeDataSet` so the previous condition should work just fine.